docs: integrate ix-icons v3 (#1600)
siemens/ix: packages/documentation/docs/icon-library/_icon_code.md

## Development

The package ```@siemens/ix-icons``` offers a large set of icons.
It also comes with the ```ix-icon``` component that displays them in your application.
Additionally custom SVG icons (that are not part of the library) can be used.

As of iX version 3.0.0 not all available iX icons will be loaded automatically anymore.

> Only the icons actually used in an app will be part of the bundle to save bandwidth and memory.
> Therefor icons need to be imported explicitly.

Reviewer: ```suggestion
Only the icons actually used in the application will be part of the bundle to save bandwidth and memory.
As a result, icons must now be imported explicitly.
```

### Usage

#### Angular

Copy iX icons into your project folder via `angular.json`.

```json
"assets": [
  "src/favicon.ico",
  "src/assets",
  {
  "glob": "**/*.svg",
  "input": "node_modules/@siemens/ix-icons/svg",
  "output": "./svg"
  }
],
```

You can use the `loadIcons` function to load individual icons once during bootstrapping.

```js
import { loadIcons } from @siemens/ix-icons;

const icons = [
  'star',
  'star-filled',
  // ...
];

loadIcons(icons);

> ```

Reviewer 2: Same as in BREAKING_CHANGES.md

#### React

```html
import { iconStar } from '@siemens/ix-icons/icons';

<IxIcon name="{iconStar}" size="16"></IxIcon>
<IxIcon name="{iconStar}" size="24"></IxIcon>

> <IxIcon name="{iconStar}" size="32"></IxIcon>

Reviewer 2: Variable assignment syntax is wrong here

#### Vue

```html
import { iconStar } from '@siemens/ix-icons/icons';

<IxIcon name="{iconStar}" size="16"></IxIcon>
<IxIcon name="{iconStar}" size="24"></IxIcon>

> <IxIcon name="{iconStar}" size="32"></IxIcon>

Reviewer 2: Variable assignment syntax is wrong here
